src/Bundle/Resources/config/skeleton/Grid.tpl.php

```diff
@@ -0,0 +1,85 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+use <?= $entity->getName() ?>;
+use Sylius\Bundle\GridBundle\Builder\Action\CreateAction;
+use Sylius\Bundle\GridBundle\Builder\Action\DeleteAction;
+use Sylius\Bundle\GridBundle\Builder\Action\ShowAction;
+use Sylius\Bundle\GridBundle\Builder\Action\UpdateAction;
+use Sylius\Bundle\GridBundle\Builder\ActionGroup\BulkActionGroup;
+use Sylius\Bundle\GridBundle\Builder\ActionGroup\ItemActionGroup;
+use Sylius\Bundle\GridBundle\Builder\ActionGroup\MainActionGroup;
+use Sylius\Bundle\GridBundle\Builder\Field\DateTimeField;
+use Sylius\Bundle\GridBundle\Builder\Field\StringField;
+use Sylius\Bundle\GridBundle\Builder\Field\TwigField;
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+
+final class <?= $class_name ?> extends AbstractGrid implements ResourceAwareGridInterface
+{
+    public function __construct()
+    {
+        // TODO inject services if required
+    }
+
+    public static function getName(): string
+    {
+        return 'app_<?= strtolower($entity->getShortName()) ?>';
+    }
+
+    public function buildGrid(GridBuilderInterface $gridBuilder): void
+    {
+        $gridBuilder
+            // see https://github.com/Sylius/SyliusGridBundle/blob/master/docs/field_types.md
+<?php
+foreach ($defaultFields as $fieldname => $type) {
+    if (in_array($type, ['STRING', 'TEXT'], true)) {
+        echo "            ->addField(\n";
+        echo "                StringField::create('".$fieldname."')\n";
+        echo "                    ->setLabel('".ucfirst($fieldname)."')\n";
+        echo "                    ->setSortable(true)\n";
+        echo "             )\n";
+    }
+
+    if (str_starts_with($type, 'DATE')) {
+        echo "            ->addField(\n";
+        echo "                DateTimeField::create('".$fieldname."')\n";
+        echo "                    ->setLabel('".ucfirst($fieldname)."')\n";
+        echo "             )\n";
+    }
+
+    if ('BOOLEAN' === $type) {
+        echo "            ->addField(\n";
+        echo "                TwigField::create('".$fieldname."', 'path/to/field/template.html.twig')\n";
+        echo "                    ->setLabel('".ucfirst($fieldname)."')\n";
+        echo "             )\n";
+    }
```

Comment:
I'm not sure about this node. What is so special about boolean node, that it is handled with TwigField? I know, that this is how we do it mostly in Sylius, but the current behaviour will result in 500 error, as the template path does not exist. I would use a string field to be sure, that the generated grid can work just after the generation

Reply:
Yes you're right, we need a generated grid which work just after the genration. I agree.

Maybe I should comment this code lines just to show you can handle it with a twig template.
and maybe add a comment with the "@SyliusUi/Grid/Field/enabled.html" for Sylius/Monofony users too...

Reply:
WDYT?

Reply:
Better question.  Why doesn't this bundle provide a `BooleanField` that's usable out-of-the-box?  A boolean field is a pretty common use case, it shouldn't need a Twig field to handle rendering unless you're aiming to do some custom display logic that a "simple" configuration couldn't handle ("simple" as in options to control the output text for both states, if you want to start using badges or icons for output that's where a Twig field is better suited).

Reply:
That's a good reflexion, not for this PR, but for the future of this bundle.

Reply:
But for sure, with TwigField (if we use it for other Doctrine types) have to be code-commented to keep the grid working just after the generation.